frontends/precinct-scanner/src/screens/admin_screen.test.tsx

```diff
@@ -182,8 +189,10 @@ test('unconfigure does not eject a usb drive that is not mounted', async () => {
     </AppContext.Provider>
   );
 
-  await fireEvent.click(screen.getByText('Unconfigure Machine'));
-  await fireEvent.click(screen.getByText('Unconfigure'));
-  expect(unconfigureFn).toHaveBeenCalledTimes(1);
-  expect(ejectFn).toHaveBeenCalledTimes(1);
+  fireEvent.click(screen.getByText('Unconfigure Machine'));
+  fireEvent.click(screen.getByText('Unconfigure'));
+  await waitFor(() => {
+    expect(unconfigureFn).toHaveBeenCalledTimes(1);
+    expect(ejectFn).toHaveBeenCalledTimes(1);
+  });
```

Comment:
surprised this worked before...

Reply:
I think it just needed the delay until the next tick provided by `await`.

Reply:
hmm, just noticing that the test above in the diff doesn't need it, which makes me suspicious, since they seem very similar